### PR TITLE
[ar] Translate multiple tiny files to Arabic

### PR DESCRIPTION
**Pull request by mboukhalfa**
Translate to arabic tiny files at once
Fix part of : #37546

### CODE REVIEW

File: content/ar/docs/setup/learning-environment/_index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Like `kind`, [`minikube`](https://minikube.sigs.k8s.io/) is a tool that lets you run Kubernetes
+locally. `minikube` runs a single-node Kubernetes cluster on your personal
+computer (including Windows, macOS and Linux PCs) so that you can try out
+Kubernetes, or for daily development work.
+
+You can follow the official
+[Get Started!](https://minikube.sigs.k8s.io/docs/start/) guide if your focus is
+on getting the tool installed.
+-->
+
```
Comment (sftim) [inline content/ar/docs/setup/learning-environment/_index.md:35]:
```suggestion
```
(see lines 11 and 12)

Remember to include the redirect!

File: content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: برنامج تعليمي تفاعلي - إنشاء عنقود
+weight: 20
+headless: true
+toc_hide: true
+_build:
+  list: never
+  publishResources: false
+---
```
Comment (sftim) [inline content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html:9]: IMO, this file does not need to exist in any new localization.

File: content/ar/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html
Comment (sftim) [inline content/ar/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html:1]: IMO, this file does not need to exist in any new localization.

File: content/ar/docs/tutorials/kubernetes-basics/explore/explore-interactive.html
Comment (sftim) [inline content/ar/docs/tutorials/kubernetes-basics/explore/explore-interactive.html:1]: IMO, this file does not need to exist in any new localization.

File: content/ar/docs/tutorials/kubernetes-basics/expose/expose-interactive.html
Comment (sftim) [inline content/ar/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:1]: IMO, this file does not need to exist in any new localization.

File: content/ar/docs/tutorials/kubernetes-basics/scale/scale-interactive.html
Comment (sftim) [inline content/ar/docs/tutorials/kubernetes-basics/scale/scale-interactive.html:1]: IMO, this file does not need to exist in any new localization.

File: content/ar/docs/tutorials/kubernetes-basics/update/update-interactive.html
Comment (sftim) [inline content/ar/docs/tutorials/kubernetes-basics/update/update-interactive.html:1]: IMO, this file does not need to exist in any new localization.

File: content/ar/releases/notes.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+title: ملاحظات
+type: docs
+description: >
+  ملاحظات إصدار كوبيرنيتيس.
+sitemap:
+  priority: 0.5
+---
+
+يمكن العثور على ملاحظات الإصدار من خلال قراءة [سجل التغييرات](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG)
+الذي يطابق إصدار كوبيرنيتيس الخاص بك. عرض سجل التغييرات لـ {{< skew currentVersionAddMinor 0 >}} on
```
Comment (sftim) [inline content/ar/releases/notes.md:12]: “on”?

File: content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html (hunk 2 above)
Comment (mboukhalfa) [reply content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html:9]: Thanks team ! do you mean that it is not required as min or that it should even been translated

Comment (sftim) [reply content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html:9]: I would leave this file out entirely. If we could, we'd serve a 410 Gone error for existing localizations; new localizations wouldn't need any error page.

File: content/ar/docs/setup/production-environment/tools/kubeadm/_index.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+---
+title: "الإقلاع التمهيدي للعناقد باستخدام كيوبئيديام (kubeadm)"
```
Comment (adowair) [inline content/ar/docs/setup/production-environment/tools/kubeadm/_index.md:2]:
```suggestion
title: "الإقلاع التمهيدي للعناقيد باستخدام كيوبئيديام (kubeadm)"
```

File: content/ar/docs/setup/production-environment/turnkey-solutions.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: الحلول السحابية الجاهزة
+content_type: concept
+weight: 40
+---
+<!-- overview -->
+
+توفر هذه الصفحة قائمة بموفري الحلول السحابية المعتمدين من كوبيرنيتيس. من كل
+صفحة موفر، يمكنك معرفة كيفية تثبيت وإعداد عناقد جاهزة للإنتاج.
```
Comment (adowair) [inline content/ar/docs/setup/production-environment/turnkey-solutions.md:9]:
```suggestion
كل موفر، يمكنك معرفة كيفية تثبيت وإعداد عناقيد جاهزة للإنتاج.
```

Comment (adowair) [inline content/ar/docs/setup/production-environment/turnkey-solutions.md:9]:
```suggestion
صفحة كل، يمكنك معرفة كيفية تثبيت وإعداد عناقد جاهزة للإنتاج.
```

File: content/ar/docs/tutorials/kubernetes-basics/create-cluster/_index.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: إنشاء عنقود
+weight: 10
+---
+
+تعرف على كوبيرنيتيس {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}} و إنشاء عنقود بسيط باستخدام مينيكيوب.
```
Comment (adowair) [inline content/ar/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]:
```suggestion
تعرف على كوبيرنيتيس {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}} وإنشاء عنقود بسيط باستخدام مينيكيوب.
```

File: content/ar/releases/notes.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+title: ملاحظات
+type: docs
+description: >
+  ملاحظات إصدار كوبيرنيتيس.
+sitemap:
+  priority: 0.5
+---
+
+يمكن العثور على ملاحظات الإصدار من خلال قراءة [سجل التغييرات](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG)
+الذي يطابق إصدار كوبيرنيتيس الخاص بك. عرض سجل التغييرات لـ {{< skew currentVersionAddMinor 0 >}} على
```
Comment (adowair) [inline content/ar/releases/notes.md:12]:
```suggestion
المطابق لإصدار كوبيرنيتيس الخاص بك. عرض سجل التغييرات لـ {{< skew currentVersionAddMinor 0 >}} على
```

File: content/ar/releases/notes.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+[جيتهاب](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-{{< skew currentVersionAddMinor 0 >}}.md).
+
+بدلاً من ذلك، يمكن البحث عن ملاحظات الإصدار وتصفيتها عبر الإنترنت على: [relnotes.k8s.io](https://relnotes.k8s.io).
```
Comment (adowair) [inline content/ar/releases/notes.md:15]:
```suggestion
بدلاً من ذلك، يمكن البحث عن ملاحظات الإصدار عبر الإنترنت وتصفيتها على: [relnotes.k8s.io](https://relnotes.k8s.io).
```

**Review by adowair — COMMENTED** [review]
Thanks @mboukhalfa! no comments/objections on the file structure, and just some minor Arabic nits

File: content/ar/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html
Comment (mboukhalfa) [reply content/ar/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html:1]: Though I can find it under the Korean localization https://github.com/kubernetes/website/blob/main/content/ko/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

File: content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html (hunk 2 above)
Comment (mboukhalfa) [reply content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html:9]: same for this on exists under the Korean l10ncan we just keep it as it is tiny translation ? https://github.com/kubernetes/website/blob/main/content/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html

**Comment by adowair** [comment]
/lgtm

^ For Arabic, and I will defer to @sftim for the final approval/lgtm

File: static/_redirects [hunk 9]
```diff
@@ -439,4 +442,3 @@
 
 /docs/concepts/overview/what-is-kubernetes/      /docs/concepts/overview/ 301
 /docs/tasks/administer-cluster/verify-signed-images/     /docs/tasks/administer-cluster/verify-signed-artifacts/   301
-
```
Comment (adowair) [inline static/_redirects:442]: Is this on purpose?

Comment (mboukhalfa) [reply static/_redirects]: Oops Nope fixed

**Comment by AbdelatifAitBara** [comment]
Hi @adowair,

Could we have LGTM on this please?
Thanks.

**Comment by AbdelatifAitBara** [comment]
Hi @tengqm,

Could you please help us to approve this PR?

The Arabic localization team does not have permission to approve changes in the file `static/_redirects`. We made a minor change to it, and that's why we are currently blocked.

The PR has already LGTMed for the Arabic, but please don't hesitate if you have any questions or need any information that can help.

We really appreciate it, 
Thank you.

**Comment by sftim** [comment]
The advice in https://github.com/kubernetes/website/pull/46172#issuecomment-2108995942 is useful but not the whole story.

SIG Docs leads are usually happy to add approval, but: as well as LGTM, please add an approval (using `/approve`) from an Arabic approver.

The PR won't merge, but Prow will still note the approval and that signifies to leads that it's OK to make the Arabic changes that are part of this PR. I don't think any SIG leads speak Arabic.

**Comment by mboukhalfa** [comment]
> SIG Docs leads are usually happy to add approval, but: as well as LGTM, please add an approval (using `/approve`) from an Arabic approver.

@adowair already added /approve but as you know because of the external file _redirects that's out of our scope of approve the bot did not add the approve label so that's need of the leads to approve 
https://github.com/kubernetes/website/pull/46172#issuecomment-2108995942

**Comment by sftim** [comment]
D'oh! You're right.

**Comment by mboukhalfa** [comment]
@sftim Can please help approve since the Ar community cannot approve changes in the file `static/_redirects`

**Comment by sftim** [comment]
> @sftim Can please help approve since the Ar community cannot approve changes in the file `static/_redirects`

You've got the wrong contributor I'm afraid. With the mechanism we use today, you should ask a tech lead or SIG chair to approve this change.

**Comment by mboukhalfa** [comment]
@natalisucks @salaxander Can you help with this we are blocked

**Comment by natalisucks** [comment]
Given reviews from @sftim and at least one Arabic owner approving these changes (@adowair), I'm adding my approval as a SIG Co-Chair to unblock this localization work 🤝 
/approve

**Comment by natalisucks** [comment]
@mboukhalfa @adowair In future, please feel free to use the alias @kubernetes/sig-docs-leads to ping all tech leads/chairs when you need an admin-level change. You can also ping our subproject owners @seokho-son and @a-mccarthy who can help triage 😸

**Pull request merged**